### PR TITLE
Edit commands behavior and add a command list

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by samuelWilliams99 — APPROVED** [review]
Looks good to me boss

File: gamemode/modules/commands/sv_commands.lua [hunk 1]
```diff
@@ -246,10 +252,32 @@ end
...
+        local plyUserGroup = RVR.getUserGroup( ply )
+
+        ply:PrintMessage( HUD_PRINTCONSOLE, "----- RaftVRaft Commands -----" )
+
+        for commandName, commandData in pairs( commands.commands ) do
+            if plyUserGroup < commandData.userGroup then continue end
+
+            local description = commandName .. ":\n" ..  commandData.description .. "\n "
+
+            ply:PrintMessage( HUD_PRINTCONSOLE, description )
```
Comment (plally) [inline gamemode/modules/commands/sv_commands.lua:274]:
```suggestion
            local shouldPrint = plyUserGroup < commandData.userGroup

            local description = commandName .. ":\n" ..  commandData.description .. "\n "

            if shouldPrint then ply:PrintMessage( HUD_PRINTCONSOLE, description ) end
```

Comment (ghost) [reply gamemode/modules/commands/sv_commands.lua:274]: Waste of computational power and the end line is too long

Comment (samuelWilliams99) [reply gamemode/modules/commands/sv_commands.lua:274]: You really gonna come out here and compute the description potentially unnecessarily?
Are you actually insane?
I'm not sure I can tollerate this kind of behaviour in this repo.
Please take a long hard think about your actions.

I'll be talking to your manager.

Comment (plally) [reply gamemode/modules/commands/sv_commands.lua:274]: I can almost guarantee the extra computation wont be an issue unless you are literally using  a potato for a computer

the line for the description is also longer

Comment (samuelWilliams99) [reply gamemode/modules/commands/sv_commands.lua:274]: you're hiding the extra scope by making it single line, but that extra scope still exists
and it'll haunt you in your dreams

File: gamemode/modules/commands/sv_commands.lua [hunk 2]
```diff
@@ -217,12 +223,12 @@ local function onPlayerSay( ply, text )
...
         ply:ChatPrint( msg )
     end
-
-    if not validCommand then
-        ply:ChatPrint( "Command \"" .. command .. "\" does not exist" )
-    end
     
-    return ""
+    if validCommand then
+    	return ""
```
Comment (plally) [inline gamemode/modules/commands/sv_commands.lua]: spacing looks  weird here

**Review by plally — CHANGES_REQUESTED** [review]

**Pull request merged**